scripts/sonic_sku_create.py

```diff
@@ -0,0 +1,853 @@
+		try:
+			shutil.copytree(self.base_sku_dir, self.new_sku_dir)
+		except OSError as e:
+			print (e.message, file=sys.stderr)
+	
+	def remove_sku_dir(self) :
+		# remove SKU directory 
+		DEFAULT_BASE_PATH = self.default_sku_path + '/' + "ACS-MSN2700/"
+		if (self.new_sku_dir in [self.base_sku_dir,DEFAULT_BASE_PATH]) :
+			print ("Removing the base SKU" + self.new_sku_dir + " is not allowed", file=sys.stderr)
+			exit(1)
+		try:
+			if not os.path.exists(self.new_sku_dir):
+				print ("Trying to remove a SKU "+ self.new_sku_dir + " that doesn't exists, Ignoring -r command")
+			while True:
+				answer = raw_input("You are about to permanently delete the SKU "+ self.new_sku_dir+" !! \nDo you want to continue (Yes/No)?") 
+				if (answer == "Yes" or answer == "No"):
+					break
+				else:
+					print("Valid answers are Yes or No")
+			if (answer == "Yes") :
+				shutil.rmtree(self.new_sku_dir)
+				print ("SKU directory: "+ self.new_sku_dir + " was removed")
+			else :
+				print ("SKU directory: "+ self.new_sku_dir + " was NOT removed")
+		except OSError as e:
+			print(e.message, file=sys.stderr) 
+					
+	def platform_specific(self) :
+                print("MADHAN platform ", self.platform)
+		func = self.platform_specific_dict.get(self.platform, lambda: "nothing")
+		return func()
+		
+		
+	def msn2700_specific(self) :
+                print ("MADHAN inside MSN2700 specific ")
+		for fp, values in self.fpp_split.items():
+			splt_arr = sorted(values[0])
+			idx_arr = sorted(values[1])
+			splt = len(splt_arr)
+			fp=fp
+			try :
+				if ((fp%2) == 1 and splt == 4) :
+					next_fp = fp+1
+					if (next_fp not in self.fpp_split) :
+						continue
+					next_fp_idx_arr = sorted(self.fpp_split[next_fp][1])
+					for i in next_fp_idx_arr :
+						if (self.verbose) :
+							print ("msn2700_specific -> Removing ", self.portconfig_dict[i])
+						self.portconfig_dict.pop(i)
+					print ("MSN2700 - Front panel port ",next_fp, " should be removed due to port ",fp,"Split by 4")
+					raise  ValueError()
+				elif ((fp%2) == 0 and splt == 4) :
+					print ("MSN2700 -  even front panel ports (",fp,") are not allowed to split by 4")
+					raise  ValueError()
+			except ValueError:
+				print ("Error - Illegal split by 4 ", file=sys.stderr)
+				exit(1)
+
+		
+	
+	def l2_mode(self,l2_sku) :
+		if not os.path.exists(self.default_sku_path+"/"+l2_sku):
+			print ("Error - path:", l2_sku, " doesn't exist", file=sys.stderr)
+			exit(1)
+		print( "Starting L2 mode configuration based on "+l2_sku+" SKU")	
+		cfg_db_str = "\n{\n\t\"DEVICE_METADATA\": {\n\t\t\"localhost\": {\n\t\t\t\"hostname\": \"sonic\"\n\t\t}\n\t}\n}\n"
+
+		subprocess.call("cat <<EOF | sudo config reload /dev/stdin -y" + cfg_db_str, shell=True)
+		subprocess.call("sudo sonic-cfggen -H --write-to-db ", shell=True)
+		subprocess.call("sonic-cfggen -t /usr/local/lib/python2.7/dist-packages/usr/share/sonic/templates/l2switch.j2 -p -k" + l2_sku + " | sudo config load /dev/stdin -y", shell=True ) 
```

Comment:
@madhanmellanox can we use sonic-cfggen --preset l2?

Reply:
removed the l2_mode_only option. Addressed in next commit.